src/azure-cli/azure/cli/command_modules/appconfig/_kv_helpers.py

```diff
@@ -281,6 +281,19 @@ def __write_kv_and_features_to_file(file_path, key_values=None, features=None, f
         raise CLIError("Failed to export key-values to file. " + str(exception))
 
 
+# Exported in the format @Microsoft.AppConfiguration(Endpoint=<storeEndpoint>; Key=<kvKey>; Value=<kvValue>; Label=<kvLabel>).
+# Label is optional
+
+def __map_to_appservice_config_reference(key_value, endpoint):
+    if endpoint is None:
+        raise CLIError("No endpoint provided.")
+
+    label = key_value.label
```

Comment:
What is the exact value of `key_value.label` when a key-value has no label? Is it `None`?

Reply:
Yes, it is.